client/src/components/APIDataTypes.ts

```diff
@@ -20,23 +20,25 @@ export type GeoSearchData = {
   bbl: string;
 };
 
-type AddressRecord = {
+export type AddressRecord = {
   bbl: string;
   bin: string;
   boro: Borough;
   businessaddrs: string[] | null;
   corpnames: string[] | null;
   evictions: number | null;
   housenumber: string;
-  lastregistrationdate: Date;
+  lastregistrationdate: string;
```

Comment:
One thing that might help here for documentation is to create a type called e.g. `APIDate` that is just a string, but where you document the type as adhering to a particular format, e.g. YYYY-MM-DD.  For example, [tenants2 has this](https://github.com/JustFixNYC/tenants2/blob/f9247d9666973fac5c2a28d0b9e53eaefd58059a/frontend/lib/networking/custom-graphql-scalars.d.ts#L1-L5).

Reply:
Nice! That's great.